app/index.js

```diff
@@ -4,14 +4,20 @@ var chalk = require('chalk');
 var yosay = require('yosay');
 var fs = require('fs');
 var request = require( 'request' );
-var async = require( 'async' );
```

Comment:
Are we sure async isn't necessary?  I seem to recall an issue related to setting the "tested up to" WP version number pull from .org.